lib/guardian.ex

```diff
@@ -190,6 +190,50 @@ defmodule Guardian do
     end
   end
 
+
+  @doc """
+  Exchange a token with type 'refresh' for a token with type 'token'
+  with the same claims but a shorter expiry time.
+
+  The token with typ 'refresh' will be revoked after the exhange
```

Comment:
Is this the normal proceedure to revoke the refresh token? I thought these stayed valid and could be used multiple times.